test/dotnet-new.Tests/DotnetNewDetailsTest.Approval.cs

```diff
@@ -177,5 +185,32 @@ public Task CanDisplayDetails_InstalledPackage_FolderInstallation()
             return Verify(commandResult.StdOut)
                 .AddScrubber(output => output.ScrubAndReplace(basicFSharp, "%TEMPLATE FOLDER%"));
         }
+
+        private async Task<string> GetLatestVersion(string packageName)
+        {
+            using (HttpClient client = new HttpClient())
+            {
+                string json = await client.GetStringAsync($"https://api.nuget.org/v3-flatcontainer/{packageName}/index.json");
```

Comment:
nuget API docs say package name should be lower-cased

```suggestion
                string json = await client.GetStringAsync($"https://api.nuget.org/v3-flatcontainer/{packageName.ToLowerInvariant()}/index.json");
```